cost-analyzer/values.yaml

```diff
@@ -205,6 +205,50 @@ kubecost:
     #  cpu: "100m"
     #  memory: "256Mi"
 
+# Kubecost Metrics deploys a separate pod which will emit kubernetes specific metrics required 
+# by the cost-model. This pod is designed to remain active and decoupled from the cost-model itself. 
+# However, disabling this service/pod deployment will flag the cost-model to emit the metrics instead. 
+kubecostMetrics: 
+  # emitPodAnnotations: false 
+  # emitNamespaceAnnotations: false 
+  # emitKsmV1Metrics: true
+  
+  # Optional
+  # The metrics exporter is a separate deployment and service (for prometheus scrape auto-discovery)
+  # which emits metrics cost-model relies on. Enabling this deployment also removes the KSM dependency
+  # from the cost-model. If the deployment is not enabled, the metrics will continue to be emitted from
+  # the cost-model.
+  exporter:
+    enabled: false
+    image: "gcr.io/kubecost1/kube-metrics:v0.0.3-SNAPSHOT"
```

Comment:
Will likely want to cut a release and change this to 0.0.3

Reply:
Do we have internal documentation on how to build this? Haven't checked out the other PRs yet.

Reply:
No, we don't. Although, I did commit the `Dockerfile`: https://github.com/kubecost/cost-model/blob/develop/Dockerfile.metrics

I have a lot of documentation to get out the door, but honestly, this is going to involve so quickly that it's going to be difficult to keep it up. I do not plan on piggy backing our on-prem product install for long.